src/Controller/AgendaBroadcastController.php

```diff
@@ -79,10 +79,10 @@ public function broadcastAgenda(Agenda $agenda, CprHelper $cprHelper, DigitalPos
                 } catch (CprException $e) {
                     switch (true) {
                         case $e->getPrevious() instanceof NoPnrFoundException:
-                            $this->addFlash('error', $translator->trans('PNR for {boardMember} not found ({cprNumber})', ['boardMember' => $boardMember->getName(), 'cprNumber' => $boardMember->getCpr()], 'digital_post'));
+                            $this->addFlash('danger', $translator->trans('PNR for {boardMember} not found ({cprNumber})', ['boardMember' => $boardMember->getName(), 'cprNumber' => $boardMember->getCpr()], 'digital_post'));
                             break;
                         case $e->getPrevious() instanceof ServiceException:
-                            $this->addFlash('error', $e->getPrevious()->getMessage());
+                            $this->addFlash('danger', $e->getPrevious()->getMessage());
                             break;
```

Comment:
You could consider using a `match` expression (https://www.php.net/manual/en/control-structures.match.php) to get the message, e.g..

```php
$message = match (true) {
  $e->getPrevious() instanceof NoPnrFoundException => $translator->trans('PNR for {boardMember} not found ({cprNumber})', ['boardMember' => $boardMember->getName(), 'cprNumber' => $boardMember->getCpr()], 'digital_post'),
  $e->getPrevious() instanceof ServiceException => $e->getPrevious()->getMessage(),
  default => $e->getMessage(),
};
$this->addFlash('danger', $message);
```

Reply:
👍